content/docs/zero/billing.md

```diff
@@ -0,0 +1,164 @@
+
+If you’re using a personal account, you must upgrade to increase quotas.
+
+## How billing works
+
+Pomerium charges your account monthly based on usage during a billing period. The billing period starts at the beginning of a calendar month and ends at the end of the month.
+
+Pomerium starts monitoring your usage when you upgrade and charges you for total usage at the end of the billing period.
+
+### Usage
+
+When you upgrade, you can choose a monthly or annual rate:
+
+- $9 per user/month (billed monthly)
+- $7 per user/month (billed annually)
+
+Pomerium calculates usage based on the total number of monthly active users (MAU) during a billing period. An “active user” includes Pomerium Zero users and Pomerium Core users.
+
+:::info How Pomerium counts unique users
+
+An admin user signing into the Zero dashboard counts as a user in Pomerium Zero. Any user accessing a Pomerium-managed route counts as a user in Pomerium Core.
+
+If a user in Zero and Core shares the same user ID, it counts as one user.
+
+:::
+
+Pomerium reports active users daily. At the end of the billing period, Pomerium adds the number of active users for the month and multiplies the total against the billing rate.
+
+For example:
+
+Usage = 200 MAU
+
+Billing rate = $7
+
+Usage x Billing rate = $1,400
+
+Based on usage, your monthly invoice would be $1,400.
+
+:::tip
+
+You can monitor MAU in the **Usage** dashboard in Pomerium Zero:
+
+![Monitor monthly active users in the Pomerium Zero Usage dashboard](./img/billing/zero-usage-dash.png)
+
+:::
+
+## Manage subscriptions
+
+You can manage subscriptions in the Zero billing dashboard.
+
+To get to the billing dashboard:
+
+1. In the navigation bar, select the account avatar.
+1. In the dropdown menu, select **Organization**.
+1. In the **Organization Settings** page, select the **Billing** tab.
+1. Select **Manage Subscription**.
+
+From here, you can:
+
+- [Upgrade your subscription](#upgrade-subscription) to a Business plan.
+- [Downgrade your subscription](#downgrade-subscription) to a Free plan.
+- Review your payment information and billing history.
+
+### Upgrade subscription
+
+When you upgrade your subscription from a Free plan to a Business plan, your account type is converted from a personal organization to a professional organization with professional quotas.
+
+To upgrade your account in Pomerium Zero, you must exceed a quota. When you do, Pomerium presents a pay wall where you can choose to upgrade your subscription.
+
+![Selecting a plan in the Pomerium Zero pay wall](./img/billing/zero-pay-wall.png)
+
+Alternatively, you can upgrade your subscription on the [Pricing](https://www.pomerium.com/pricing) page.
+
+Upgrading does not replace your personal account. It creates an additional professional organization separate from your personal account.
```

Comment:
All the existing clusters in the personal account will be moved to the professional account... not sure the best way to say that.

Reply:
For context reasons, I'm restating our conversation. We decided not to include this note about cluster migration form a personal account to a professional org account. The reason is that we limit personal accounts to 1 cluster. In my experience, my personal account retained the configuration and the new professional cluster was empty. This may be because my account was available before a lot of this functionality was merged, but I could be wrong.